Makefile

```diff
@@ -0,0 +1,17 @@
+
+.PHONY: depinstall clean sdist test
+
+clean:
```

Comment:
This will make the default `make` target `clean` - it should be `build` (which in this case is what's currently `depinstall`)